po/fr.po

```diff
@@ -276,8 +277,8 @@ msgid ""
 "All windows except for the peeked one have their opacity set to the same "
 "value."
 msgstr ""
-"Toute fenêtre, mis à part celle prévisualisée, ont leur opacité mise à la "
-"même valeur."
+"Toute les fenêtres, mis à part celle prévisualisée, ont leur opacité mise à "
```

Comment:
Thanks for all the updates!

Still, a small missing `s` over here :). `Toutes`.

Merci!